giraph-core/src/main/java/org/apache/giraph/ooc/policy/MemoryEstimatorOracle.java

```diff
@@ -792,7 +792,6 @@ private static boolean calculateRegression(double[] coefficient,
         LOG.warn("There are " + coefficient.length +
```

Comment:
We should make this a `LOG.info`. We can also entirely remove this `if` block as it doesn't add much info to the log actually. If we want to keep the `if`, we should also remove the "but" from the logline :-)